config/configmap/inferenceservice.yaml

```diff
@@ -199,5 +199,6 @@ data:
     }
   deploy: |-
     {
-      "defaultDeploymentMode": "Serverless"
+      "defaultDeploymentMode": "Serverless",
+      "istioDisabled": true
```

Comment:
Let's keep this flag in the ingress config, I think the better name is `disableIstioVirtualHost`.